testing/test_skipping.py

```diff
@@ -713,6 +714,27 @@ def doskip():
     assert result.ret == 0
 
 
+def test_skipped_folding(testdir):
+    testdir.makepyfile(
```

Comment:
Thanks for fast reply!

What may I do now? Add comments about workaround, or should we postpone this PR until marks would be fixed?